iaso/models/org_unit.py

```diff
@@ -341,6 +345,9 @@ def calculate_paths(self, force_recalculate: bool = False) -> typing.List["OrgUn
     def __str__(self):
         return "%s %s %d" % (self.org_unit_type, self.name, self.id if self.id else -1)
 
+    def create_ou_tree(self):
```

Comment:
Can we add a docstring explaining what it does, a type annotation for a return value and a unit test to make sure it works (and continue to work in the future)?